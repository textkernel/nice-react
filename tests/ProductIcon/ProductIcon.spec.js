import React from 'react';
import { ProductIcon } from '../../dist';

describe('<ProductIcon />', () => {
    it('should render nothing', () => {
        const wrapper = shallow(<ProductIcon />);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render an icon', () => {
        const wrapper = shallow(<ProductIcon product="jobfeed" />);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render with extra classes and styles', () => {
        const wrapper = shallow(<ProductIcon product="jobfeed" className="text-color-brand" style={{ fontSize: '3em' }} />);
        expect(wrapper).toMatchSnapshot();
    });
});
