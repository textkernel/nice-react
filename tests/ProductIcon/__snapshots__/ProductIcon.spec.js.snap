// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductIcon /> should render an icon 1`] = `
<i
  className="product-icon jobfeed "
  style={null}
/>
`;

exports[`<ProductIcon /> should render nothing 1`] = `""`;

exports[`<ProductIcon /> should render with extra classes and styles 1`] = `
<i
  className="product-icon jobfeed text-color-brand"
  style={
    Object {
      "fontSize": "3em",
    }
  }
/>
`;
